### PR TITLE
Add possibility to check contracts in a batch

### PR DESCRIPTION
**Pull request by folkyatina**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -121,5 +104,29 @@ def main():
...
+
+    logger.divider()
+
+    logger.info("Removing artifacts from the previous run...")
+    remove_directory(DIGEST_DIR)
+
+    logger.info("Loading config...")
+    config = load_config()
+
+    if contract_address is not None:
```
Comment (TheDZhon) [inline main.py:124]: let's log the chosen mode explicitly

File: config.sample.json [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+        "0x710B3303fB508a84F10793c1106e32bE873C24cd",
+        "0x388C818CA8B9251b393131C08a736A67ccB19297"
+    ],
     "network": "mainnet",
     "github_repo": "https://github.com/lidofinance/lido-dao/tree/master",
     "dependencies": {
-        "@aragon": "https://github.com/aragon/aragonOS/tree/f3ae59b00f73984e562df00129c925339cd069ff",
-        "openzeppelin-solidity": "https://github.com/OpenZeppelin/openzeppelin-contracts/tree/06e265b38d3e9daeaa7b33f9035c700d6bc0c6a0",
-        "solidity-bytes-utils": "https://github.com/GNSPS/solidity-bytes-utils/tree/9776282d181839fbb4b18f2cf218e316d6df871c"
+        "@aragon": "https://github.com/aragon/aragonOS/tree/v4.4.0",
```
Comment (TheDZhon) [inline config.sample.json:11]: We use `4.4.1`
https://github.com/lidofinance/lido-dao/blob/master/package.json#L120

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM except only a few things :+1:

**Review by mymphe — APPROVED** [review]

**Comment by mymphe** [comment]
LGTM

**Pull request merged**